[2022-05-23 20:16:28,966] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.thrid_task scheduled__2022-04-14T00:00:00+00:00 [queued]>
[2022-05-23 20:16:28,986] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.thrid_task scheduled__2022-04-14T00:00:00+00:00 [queued]>
[2022-05-23 20:16:28,986] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 20:16:28,986] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-05-23 20:16:28,986] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 20:16:29,073] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): thrid_task> on 2022-04-14 00:00:00+00:00
[2022-05-23 20:16:29,098] {standard_task_runner.py:52} INFO - Started process 3139 to run task
[2022-05-23 20:16:29,122] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'thrid_task', 'scheduled__2022-04-14T00:00:00+00:00', '--job-id', '784', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp_kxenmel', '--error-file', '/tmp/tmpuhmxx47c']
[2022-05-23 20:16:29,123] {standard_task_runner.py:80} INFO - Job 784: Subtask thrid_task
[2022-05-23 20:16:29,409] {task_command.py:369} INFO - Running <TaskInstance: our_first_dag_v5.thrid_task scheduled__2022-04-14T00:00:00+00:00 [running]> on host 061a475e6c14
[2022-05-23 20:16:29,617] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=gerson
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=thrid_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-14T00:00:00+00:00
[2022-05-23 20:16:29,618] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-23 20:16:29,618] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task3 and will be running after task1 at the same time as task2!']
[2022-05-23 20:16:29,652] {subprocess.py:85} INFO - Output:
[2022-05-23 20:16:29,658] {subprocess.py:92} INFO - hey, I am task3 and will be running after task1 at the same time as task2!
[2022-05-23 20:16:29,659] {subprocess.py:96} INFO - Command exited with return code 0
[2022-05-23 20:16:29,773] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=thrid_task, execution_date=20220414T000000, start_date=20220523T201628, end_date=20220523T201629
[2022-05-23 20:16:29,846] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-23 20:16:29,903] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
